src/cudamatrix/cu-device.h

```diff
@@ -83,6 +84,7 @@ class CuDevice {
   inline cublasHandle_t GetCublasHandle() { return cublas_handle_; }
   inline cusparseHandle_t GetCusparseHandle() { return cusparse_handle_; }
   inline curandGenerator_t GetCurandHandle() { return curand_handle_; }
+  inline cusolverDnHandle_t GetCusolverDnHandle() { return cusolverdn_handle_; }
```

Comment:
Is this needed for something just yet?  Not that I necessarily object, just asking.